manifests/params.pp

```diff
@@ -140,6 +152,12 @@
     validate_re($rhn_password, '.+')
   }
 
+  if $rhnget_cleanup != undef {
```

Comment:
Maybe I take that back.  I suppose if it is `false` there's no real need to hit the `validate_bool`.  But it might be safer to leave as is in case any other code is ever added to the if block?